config/cmd/cmd.go

```diff
@@ -650,6 +675,18 @@ func (c *cmd) Before(ctx *cli.Context) error {
 
 	// Setup registry options
 	registryOpts := []registry.Option{registrySrv.WithClient(microClient)}
+
+	// Parse registry TLS certs
+	if ctx.Bool("registry_secure") {
+		cert, err := tls.LoadX509KeyPair("/certs/registry/cert.pem", "/certs/registry/key.pem")
```

Comment:
In a future version I wonder whether we want this configurable too.

Reply:
I think these should probably be configurable or loaded from a relative path.